pkg/codegen/configuration.go

```diff
@@ -35,6 +35,7 @@ type GenerateOptions struct {
 	Client        bool `yaml:"client,omitempty"`         // Client specifies whether to generate client boilerplate
 	Models        bool `yaml:"models,omitempty"`         // Models specifies whether to generate type definitions
 	EmbeddedSpec  bool `yaml:"embedded-spec,omitempty"`  // Whether to embed the swagger spec in the generated code
+	NullableType  bool `yaml:"nullable-type,omitempty"`  // Whether to generate nullable type for nullable fields
```

Comment:
Sorry, would this be able to move to the `OutputOptions`? That's a better place for it